frontend/packages/pydantic-forms/src/components/fields/ArrayField.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { useFieldArray } from 'react-hook-form';
+
+import { usePydanticFormContext } from '@/core';
+import { fieldToComponentMatcher } from '@/core';
+import { PydanticFormElementProps } from '@/types';
+import { itemizeArrayItem } from '@/utils';
+
+import { RenderFields } from '../render';
+
+export const ArrayField = ({ pydanticFormField }: PydanticFormElementProps) => {
+    const { rhf, config } = usePydanticFormContext();
+    const { control } = rhf;
+    const { id: arrayName, arrayItem } = pydanticFormField;
+    const { minItems, maxItems } = pydanticFormField.validations;
+    const { fields, append, remove } = useFieldArray({
+        control,
+        name: arrayName,
+    });
+    if (!arrayItem) return '';
+
+    const component = fieldToComponentMatcher(
+        arrayItem,
+        config?.componentMatcher,
+    );
+
+    return (
+        <div
+            style={{
+                border: 'thin solid green',
+                padding: '1rem',
+                marginTop: '16px',
+                display: 'flex',
+                flexDirection: 'column',
+                flexGrow: 1,
```

Comment:
This applies to multiple inline styles in the project/PR: I think we should start applying some way of standardization of styles or coding styles, like we did in the orchestrator-ui. Especially when we start using this in production